packages/react/src/Testimonial/Testimonial.module.css

```diff
@@ -18,18 +21,36 @@
 }
 
 .Testimonial--size-large {
+  --testimonial-quote-size: var(--brand-text-size-700);
   gap: var(--base-size-24);
 }
 
+.Testimonial--size-large::before {
+  font-size: 3em;
+  line-height: 1;
+}
+
 .Testimonial-quote {
+  font-size: 1em;
   font-family: var(--brand-fontStack-sansSerif);
-  font-size: var(--brand-text-size-300);
   margin-top: calc(var(--base-size-16) * -1); /* negate grid gap */
   display: block;
 }
 
 .Testimonial--size-large .Testimonial-quote {
-  font-size: var(--brand-text-size-700);
+  margin-top: calc(var(--base-size-24) * -1); /* negate grid gap */
+}
+
+.Testimonial-quote:has(em) {
```

Comment:
`has` doesn't have comprehensive browser support yet, so could you handle this inside the component for the time being? [Example of `em` support added to section intro using `child.type`](https://github.com/primer/brand/pull/376/files#diff-dcc5852ace9baf504d6890f39cf2a451591d81598cbdcc9bcb4f716c8a74f69fR51-R60).

Reply:
Falling back to a single color seems reasonable for this. Curious what other folks think.